app/screens/emoji_picker/picker/sections/index.tsx

```diff
@@ -150,7 +155,48 @@ const EmojiSections = ({customEmojis, customEmojisEnabled, onEmojiPress, recentE
                 key: category,
             };
         }).filter((s: EmojiSection) => s.data.length);
-    }, [customEmojis, customEmojisEnabled, isTablet]);
+
+        if (imageUrl || file) {
+            sectionsArray.unshift({
+                data: [[{
+                    aliases: [],
+                    name: imageUrl || file?.name || 'file',
+                    short_name: imageUrl || file?.name || 'file',
+                    category: 'image',
+                }]],
+                defaultMessage: 'Default',
+                icon: 'file-image-outline',
+                id: 'emoji_picker.default',
+                key: 'default',
+                renderItem: ({item}: ListRenderItemInfo<EmojiAlias[]>) => {
+                    return (
+                        <View style={styles.row}>
+                            <View style={styles.emoji}>
+                                <TouchableWithFeedback onPress={() => onEmojiPress('')}>
+                                    <>
+                                        {Boolean(file) &&
+                                        <FileIcon
+                                            file={file as FileInfo}
+                                            iconSize={30}
+                                        />
+                                        }
+                                        {Boolean(imageUrl) &&
+                                        <FastImage
+                                            source={{uri: item[0].name}}
+                                            style={{width: 28, height: 28}}
+                                        />
+                                        }
+                                    </>
+                                </TouchableWithFeedback>
+                            </View>
+                        </View>
```

Comment:
We could consider extracting this to a component, so we can do, among other things, a `useCallback` on the Touchable `onPress`.